scripts/abundance-dist.py

```diff
@@ -95,6 +98,10 @@ def main():
         sys.exit(1)
     hash_fp = open(args.output_histogram_filename, 'w')
 
+    if args.Header_names:
+        print("k-mer_abundance", "K-mer_count", "Cumulative_count" , 
```

Comment:
Hmm, this is python3.0 format for print, I think? Didn't realize it worked in python 2.7 :).  Should be

print >> hash_fp, ...

to be in line with the rest of the codebase. Not sure how important this is?